scripts/generateComponent.js

```diff
@@ -0,0 +1,45 @@
+// e.g. yarn generate Foo
+
+const fs = require('fs-extra');
+const argv = require('yargs').argv;
+const replace = require('replace-in-file');
+const Renamer = require('renamer');
+const renamer = new Renamer();
+
+const componentName = argv._[0];
+
+if (componentName) {
```

Comment:
In addition to checking if an argument was passed for the component name, it might also be good to have a check to ensure that the component doesn't already exist. Otherwise this script will still generate files in the destination folder. Attaching a screenshot of the two components I made, then running the script again. 

![image](https://user-images.githubusercontent.com/2113956/64578425-f8633980-d344-11e9-99c3-059f3dc89ddf.png)